dev/com.ibm.etools.maven.liberty.integration/src/com/ibm/etools/maven/liberty/integration/internal/LibertyMaven.java

```diff
@@ -187,7 +187,7 @@ public boolean isSupportedProject(IProject project, IProgressMonitor monitor) {
     /** {@inheritDoc} */
     @Override
     public void updateSrcConfig(IPath location, LibertyBuildPluginConfiguration config, IProgressMonitor monitor) {
-        String goal = "package liberty:install-apps";
+        String goal = "package liberty:deploy";
```

Comment:
Don't you need to check which version of the liberty maven plugin is configured before deciding whether to call install-apps or deploy goal? Before 3.x, install-apps was the correct goal to call. Calling deploy instead would not work if a 2.x version of the plugin was configured.

Reply:
Please consider also the other changes like create-server -> create and so on ...

Reply:
> Please consider also the other changes like create-server -> create and so on ...


Yes, they will be handled together.

Reply:
> Don't you need to check which version of the liberty maven plugin is configured before deciding whether to call install-apps or deploy goal? Before 3.x, install-apps was the correct goal to call. Calling deploy instead would not work if a 2.x version of the plugin was configured.

This update will result in some impact on Websphere Developer Tools as well, so handling both 2.x/3.x may not be an easy option. Rather, we should consider to document it to use 3.x for upcoming OLT/WDT releases (20.0.0.3 and higher)